crates/pallet-feeds/src/lib.rs

```diff
@@ -181,6 +184,14 @@ mod pallet {
         }};
     }
 
+    #[pallet::hooks]
+    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+        fn on_initialize(_now: BlockNumberFor<T>) -> Weight {
+            SuccessfulCalls::<T>::take();
+            T::DbWeight::get().reads_writes(1, 1)
```

Comment:
Maybe writing empty vector instead? Then we don't need to read it, only override at the end of the block processing.

`kill()` will have the same effect most likely.

Reply:
true. Kill it is